### DISCUSSION TITLE
Cannot specify both 'width' and 'min-width'

### DISCUSSION THREAD

**Discussion by iincity** [syf20020816/SurrealismUI#42] [General]
when use widget SInput
error: Cannot specify both 'width' and 'min-width'
surrealism-ui\src\card\card.slint:50:11
width : UseSurrealismFn.count-width(self.card-width,self.padding-left);

**Comment by syf20020816** [comment]
`min-width` can not use with `width` together, this is something you need to pay attention to
It is not an error

**Reply by iincity** [reply]
我只是想找到一个途径去解决，并不是说这个是一个BUG

**Comment by iincity** [comment]
整个工程中都不能使用？

**Reply by syf20020816** [reply]
you can remove the `min-width` set in source code, so that you can do that

**Comment by iincity** [comment]
我尝试移除过，但是结果无法显示出来。现在还在测试中

**Reply by syf20020816** [reply]
BTW, `min-width` is in SCard

**Reply by iincity** [reply]
![image](https://github.com/Surrealism-All/SurrealismUI/assets/5435797/2d82a75f-b25a-4dfe-9748-d0c5dace2aae)
![image](https://github.com/Surrealism-All/SurrealismUI/assets/5435797/5fbc8bdb-9984-4ae3-bfab-b3ecf40ee364)

**Comment by syf20020816** [comment]
We have discussed this before：[url](https://github.com/Surrealism-All/SurrealismUI/discussions/18)
